tuf-spec.md

```diff
@@ -1425,14 +1445,34 @@ it in the next step.
   in the trusted timestamp metadata.  If the versions do not match, discard the
   new snapshot metadata, abort the update cycle, and report the failure.
 
-5. **Check for a rollback attack**. The version number of the targets
-  metadata file, and all delegated targets metadata files, if any, in the
-  trusted snapshot metadata file, if any, MUST be less than or equal to its
-  version number in the new snapshot metadata file. Furthermore, any targets
-  metadata filename that was listed in the trusted snapshot metadata file, if
-  any, MUST continue to be listed in the new snapshot metadata file.  If any of
-  these conditions are not met, discard the new snapshot metadata file, abort
-  the update cycle, and report the failure.
+5. **Check for a rollback attack**.
+
+  1. The new snapshot metadata file's [=METAFILES=] object MUST contain a
+    [=snapshot/METAPATH=] entry for the targets metadata file, and all delegated targets
+    metadata files, if any, in the trusted snapshot metadata file.  If not,
+    discard the new snapshot metadata file, abort the cycle, and report the
```

Comment:
Do we need to discard the snapshot file in this case?  Why?

Reply:
This was covered in the above comment, so lets track it there.